src/components/common/TmSessionExtension.vue

```diff
@@ -76,13 +76,31 @@ export default {
       })
     },
     async signInAndRedirect(account) {
+      let accountNetwork = this.networks.find(
```

Comment:
this should reallly really really really become a getter^^

Reply:
Converted to universal getter, defaulting to id